code/go/0chain.net/blobber/http.go

```diff
@@ -38,7 +38,7 @@ func startHttpServer() {
 	go startServer(&wg, r, mode, httpsPort, true)
 
 	logging.Logger.Info("Ready to listen to the requests")
-	fmt.Println("[11/11] start http server	[OK]")
+	fmt.Println("[12/12] start http server	[OK]")
```

Comment:
Can we not have one common method which task msg string as param and handle [current-step/total-step]

Reply:
It seems be not very worth. because current-step always be changed. but total-setp can be a variables.

Reply:
I am not sure if it is worth to save a lot of time to extract it to method. Because it is not a functional method.